sql/item_mods.sql

```diff
@@ -34233,22 +34233,23 @@ INSERT INTO `item_mods` VALUES (26995,31,37);
 INSERT INTO `item_mods` VALUES (26995,68,24);
 INSERT INTO `item_mods` VALUES (26995,121,11);
 INSERT INTO `item_mods` VALUES (26995,161,-3);
-INSERT INTO `item_mods` VALUES (26995,384,30);
-INSERT INTO `item_mods` VALUES (26996,1,62);
-INSERT INTO `item_mods` VALUES (26996,2,11);
-INSERT INTO `item_mods` VALUES (26996,8,11);
-INSERT INTO `item_mods` VALUES (26996,9,22);
-INSERT INTO `item_mods` VALUES (26996,10,18);
-INSERT INTO `item_mods` VALUES (26996,12,8);
-INSERT INTO `item_mods` VALUES (26996,13,19);
-INSERT INTO `item_mods` VALUES (26996,14,11);
-INSERT INTO `item_mods` VALUES (26996,27,-3);
-INSERT INTO `item_mods` VALUES (26996,29,1);
-INSERT INTO `item_mods` VALUES (26996,31,25);
-INSERT INTO `item_mods` VALUES (26996,68,24);
-INSERT INTO `item_mods` VALUES (26996,66,13);
-INSERT INTO `item_mods` VALUES (26996,384,40);
-INSERT INTO `item_mods` VALUES (26996,423,3);
+INSERT INTO `item_mods` VALUES (26995,384,31);
+INSERT INTO `item_mods` VALUES (26996,1,62); -- Arcadian Bracers DEF 62
+INSERT INTO `item_mods` VALUES (26996,2,11); -- HP +11
+INSERT INTO `item_mods` VALUES (26996,8,11); -- STR +11
+INSERT INTO `item_mods` VALUES (26996,9,22); -- DEX +22
+INSERT INTO `item_mods` VALUES (26996,10,18); -- VIT +18
+INSERT INTO `item_mods` VALUES (26996,11,8); -- AGI +8
+INSERT INTO `item_mods` VALUES (26996,12,8); -- INT +8
+INSERT INTO `item_mods` VALUES (26996,13,19); -- MND +19
+INSERT INTO `item_mods` VALUES (26996,14,11); -- CHR +11
+INSERT INTO `item_mods` VALUES (26996,27,-3); -- Enmity -3
+INSERT INTO `item_mods` VALUES (26996,29,1); -- "Magic Def. Bonus" +1
+INSERT INTO `item_mods` VALUES (26996,31,25); -- Magic Evasion +25
+INSERT INTO `item_mods` VALUES (26996,68,24); -- Evasion +24
+INSERT INTO `item_mods` VALUES (26996,66,13); -- Ranged Attack +13
+INSERT INTO `item_mods` VALUES (26996,384,41); -- Haste +4% (4% of 1024 = 40.96 rounded up to 41)
```

Comment:
So if it's 40.(whatever decimal), the value is strictly 40?
Had to look up the definition of truncate lol.
```verb
1.
shorten (something) by cutting off the top or the end.```